# HG changeset patch
# User Dick Marinus <dick@mrns.nl>
# Date 1370008703 -7200
# Node ID 6b713ccfcda11009f82ea2de4c84d16515b3f67f
# Parent  d43bc1b83008a2f5937c15176ffc914555c4bfcb
don't match files from systemd like systemd-journald.socket

Signed-off-by: Dick Marinus <dick@mrns.nl>

diff -r d43bc1b83008a2f5937c15176ffc914555c4bfcb -r 6b713ccfcda11009f82ea2de4c84d16515b3f67f policy/libraries.py
--- a/policy/libraries.py	Wed May 29 19:15:06 2013 +0000
+++ b/policy/libraries.py	Fri May 31 15:58:23 2013 +0200
@@ -283,7 +283,7 @@
     processUnmodified = False
     invariantexceptions = [ '(%(debuglibdir)s|%(debugsrcdir)s)/', ] # CNP-201
     invariantinclusions = [
-        '.*\.(so.*|a)$',
+        '.*\.(so\..*|so|a)$',
     ]
 
     def __init__(self, *args, **keywords):
